scripts/dev_setup.py

```diff
@@ -26,6 +26,12 @@ def exec_command(command):
 print('Running dev setup...')
 print('Root directory \'{}\'\n'.format(root_dir))
 
+# install private whls if there are any
+privates_dir = os.path.join(root_dir, 'privates')
+if os.path.isdir(privates_dir) and os.listdir(privates_dir):
+    whl_list = ' '.join([os.path.join(privates_dir, f) for f in os.listdir(privates_dir)])
+    exec_command('pip install {}'.format(whl_list))
```

Comment:
Ask forgiveness instead of permission.

```
try:
    whl_list = ' '.join([os.path.join(private_dir, f) for f in os.listdir(privates_dir)])
    exec_command('pip install {}'.format(whl_list))
except OSError:
    pass
```

Reply:
Okay can make this change in next PR.

Reply:
I looked into this and this would run 'pip install <no_files>' if `privates` directory exists and there are no whls in there.
I could add an if statement to check but then it kind of defeats the point I think.